docs/how-to/native-install/includes/ubuntu-multi-install.rst

```diff
@@ -0,0 +1,75 @@
+.. _ubuntu-multi-install:
+
+1. Set up your package signing key following the steps in :ref:`ubuntu-package-key`.
+
+2. Register the kernel-mode driver.
+
+   Add the AMDGPU repository for the driver.
```

Comment:
```suggestion
   * Add the AMDGPU repository for the driver.
   
```
In the built page, I think this looks weird as 2 lines.  Can this be bulleted or perhaps collapse to 1 line instead?

Reply:
agreed. collapsed to 1 line